notebooks/07-Advanced_Topics_and_Plans.ipynb

```diff
@@ -0,0 +1,296 @@
+{
```

Comment:
<p>Bullets messed up here too?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/asdf-format/scipy2022tutorial/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>reworked this outdated cell</p>